jhotdraw-datatransfer/src/main/java/org/jhotdraw/datatransfer/ImageTransferable.java

```diff
@@ -40,18 +40,19 @@ public boolean isDataFlavorSupported(DataFlavor flavor) {
 
   @Override
   public Object getTransferData(DataFlavor flavor) throws UnsupportedFlavorException, IOException {
-    /*if (! isDataFlavorSupported(flavor)) {
-        throw new UnsupportedFlavorException(flavor);
-    }*/
+
+    Object result;
+
     if (flavor.equals(DataFlavor.imageFlavor)) {
```

Comment:
It is necessary to check null for object flavor before accessing the value to avoid a null pointer exception.

Reply:
@AyshaThasneemGulshan Ok I got it. Thanks.